_layouts/home.html

```diff
@@ -141,13 +143,13 @@ <h3 class="panel-title">Input languages</h3>
                                 <h3 class="panel-title">Properties</h3>
                             </div>
                             <div class="panel-body">
-                                Storm focuses on reachability queries and its support includes
+                                Supported model checking queries include
```

Comment:
The focus on expected rewards and reachability/reach-avoid is maybe not bad to keep explicit?